R/initialize.R

```diff
@@ -45,6 +45,19 @@ sd_initialize <- function(self, private, path, loadTimeout, checkNames,
   self$logEvent("Navigating to Shiny app")
   private$web$go(private$getShinyUrl())
 
+  "!DEBUG wait for navigation to happen"
+  nav_stop_time <- as.numeric(Sys.time()) + loadTimeout
+  while(private$web$getUrl() == "about:blank") {
```

Comment:
Just because I'm not sure we can assume >0 length?

```suggestion
  while(identical(private$web$getUrl(), "about:blank")) {
```

Reply:
Good call, thanks.